src/components/TemplateLibrary/index.js

```diff
@@ -19,6 +19,13 @@ const LibraryComponent = () => {
     load();
   }, []);
 
+  const goToTemplateDetail = template => {
+    const templateOrigin = new URL(template.url).origin;
+    const templateName = template.name;
+    const templateVersion = template.version;
```

Comment:
I think you could do 

```js
const { name, version } = template;
```

to extract those values and not need to create `templateName` or `templateVersion`